dynamiqs/utils/jax_utils.py

```diff
@@ -95,3 +81,45 @@ def to_qutip(x: ArrayLike, dims: tuple[int, ...] | None = None) -> Qobj | list[Q
         elif isop(x):  # [[3], [3]] or for composite systems [[3, 4], [3, 4]]
             dims = [dims, dims]
         return Qobj(x, dims=dims)
+
+
+def device(backend: Literal['cpu', 'gpu', 'tpu']):
+    """Configure the default device.
+
+    Notes:
+        This function is equivalent to
+        ```
+        jax.config.update('jax_default_device', jax.devices(backend)[0])
+        ```
+
+    See [JAX documentation on devices](https://jax.readthedocs.io/en/latest/faq.html#faq-data-placement).
+
+    Args:
+        backend _(string 'cpu', 'gpu', or 'tpu')_: Default device.
+    """
+    jax.config.update('jax_default_device', jax.devices(backend)[0])
+
+
+def precision(dtype: jnp.float32 | jnp.float64):
```

Comment:
I would suggest `def set_precision(precision: Literal['simple', 'double'])` (similar API to `device`), or `def set_dtype(dtype: jnp.float32 | jnp.float64)`. Here it's somewhere in between the two.